fortnite_api/cosmetics/common.py

```diff
@@ -0,0 +1,208 @@
+    display_value: :class:`str`
+        The display value of the cosmetic rarity. This is the value that is displayed to the user.
+    backend_value: :class:`str`
+        The internal marker of the cosmetic rarity.
+    """
+
+    __slots__: Tuple[str, ...] = ('value', 'display_value', 'backend_value')
+
+    def __init__(self, *, data: Dict[str, Any]) -> None:
+        self.value: CosmeticRarity = CosmeticRarity(data['value'])
+        self.display_value: str = data['displayValue']
+        self.backend_value: str = data['backendValue']
+
+
+class CosmeticSeriesInfo(Generic[HTTPClientT]):
+    """
+    .. attributetable:: fortnite_api.CosmeticSeriesInfo
+
+    Represents information about the series a :class:`~fortnite_api.Cosmetic` belongs to.
+
+    Attributes
+    ----------
+    value: :class:`str`
+        The value of the cosmetic series.
+    backend_value: :class:`str`
+        The internal marker of the cosmetic series.
+    image: Optional[:class:`fortnite_api.Asset`]
+        The image of the cosmetic series. Will be ``None`` if the cosmetic series has no image.
+    colors: List[:class:`str`]
+        A list of colors that are associated with the cosmetic series.
+    """
+
+    __slots__: Tuple[str, ...] = ('value', 'backend_value', 'image', 'colors')
+
+    def __init__(
+        self,
+        *,
+        data: Dict[str, Any],
+        http: HTTPClientT,
+    ) -> None:
+        self.value: str = data['value']
+        self.backend_value: str = data['backendValue']
+
+        image = data.get('image')
+        self.image: Optional[Asset[HTTPClientT]] = image and Asset(http=http, url=image)
+
+        self.colors: List[str] = get_with_fallback(data, 'colors', list)
+
+
+class CosmeticImages(Images[HTTPClientT]):
+    """
+    .. attributetable:: fortnite_api.CosmeticImages
+
+    Represents the images of a cosmetic.
+
+    This inherits from :class:`fortnite_api.Images`.
+
+    Attributes
+    ----------
+    featured: Optional[:class:`fortnite_api.Asset`]
+        The featured image of the cosmetic, if available.
+    lego: Optional[:class:`fortnite_api.Asset`]
+        The LEGO image of the cosmetic, if available.
+    other: Dict[:class:`str`, :class:`fortnite_api.Asset`]
+        A mapping other images to their respective asset.
+    """
+
+    __slots__: Tuple[str, ...] = ('featured', 'lego', 'other')
+
+    def __init__(
+        self,
+        *,
+        data: Dict[str, Any],
+        http: HTTPClientT,
+    ) -> None:
+        super().__init__(data=data, http=http)
+
+        featured = data.get('featured')
+        self.featured: Optional[Asset[HTTPClientT]] = featured and Asset(http=http, url=featured)
+
+        lego = data.get('lego')
+        self.lego: Optional[Asset[HTTPClientT]] = lego and Asset(http=http, url=lego)
+
+        # Mapping of str to Asset. In testing all I have found is "background" in this mapping.
+        other: Dict[str, str] = get_with_fallback(data, 'other', dict)
```

Comment:
This misses many stuff.

`CosmeticBr` usually got `smallIcon`, `icon`, `featured`, `lego` and `other`
`CosmeticInstrument`, `CosmeticCar` and `CosmeticLegoKit` usually got `small` and `large`
`CosmeticLego` usually got `small`, `large`, `wide`